Erro ao carregar o módulo do prettier

Ao clonar o repositório e digitar o comando `npm run dev`, é mostrado o seguinte erro:

```
Module build failed (from ./node_modules/eslint-loader/index.js):
Error: Failed to load plugin 'prettier' declared in '.eslintrc.js': Cannot find module 'eslin
t-plugin-prettier
```

--- a/.eslintrc.js
+++ b/.eslintrc.js
@@ -3,11 +3,7 @@ module.exports = {
   env: {
     node: true
   },
-  extends: [
-    "@vue/standard",
-    "plugin:prettier/recommended",
-    "plugin:vue/essential"
-  ],
+  extends: ["@vue/standard", "plugin:vue/essential"],
   rules: {
     "no-console": process.env.NODE_ENV === "production" ? "error" : "off",
     "no-debugger": process.env.NODE_ENV === "production" ? "error" : "off",

